soc/arm/atmel_sam0/common/Kconfig.defconfig.series

```diff
@@ -0,0 +1,35 @@
+# Kconfig.defconfig.series - Atmel SAM0 all MCU lines
+#
+# Copyright (c) 2019 Benjamin Valentin
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_FAMILY_SAM0
```

Comment:
That's a good idea but we probably don't need a separate `Kconfig.defconfig.series` file. I would put this section directly into `soc/arm/atmel_sam0/common/Kconfig.defconfig`.

Reply:
Ok, I was just copying what `st_stm32` was doing.